README.md

```diff
@@ -117,19 +139,23 @@ from https://github.com/facebook/watchman
 ## Requirements
 
 * [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* `java` and `jar` on the PATH (make sure you use `java` executable from JDK but not JRE)
+* `java` and `jar` on the PATH (make sure you use `java` executable from JDK but not JRE). 
+  * To test this, try running the command `javac`. This command won't exist if you only have the JRE
+  installed. If you're met with a list of command-line options, you're referencing the JDK properly.
+  * If you're running Java 11, you won't be able to build Selenium. We recommend using a tool like  
+  `jenv` to manage different using versions of Java 
 * [Python 2.7](https://www.python.org/)
 * `python` on the PATH (make sure it's Python 2.7, as buck build tool is not Python 3 compatible)
 * [The Requests Library](http://python-requests.org) for Python: `pip install requests`
-* MacOS users should have XCode installed
+* MacOS users should have the latest version of XCode installed, including the command-line tools
```

Comment:
`xcode-select --install` (or is it `xcode-select install`?) can be used to install the command line tools.

Reply:
It is `xcode-select --install`, @mmerrell maybe we could add it?